sixtyfps_compiler/passes/lower_layout.rs

```diff
@@ -9,6 +9,8 @@
 LICENSE END */
 //! Passe that compute the layout constraint
 
+use lyon_path::geom::euclid::approxeq::ApproxEq;
```

Comment:
I'd like to suggest to use euclid directly, instead of going through a re-export - unless there's a good reason to prefer the indirection?